Makefile

```diff
@@ -60,20 +60,20 @@ perf-debug:
 
 
 nightly-release: neard-nightly-release
-	cargo build -p store-validator --release --features nearcore/nightly_protocol,nearcore/nightly_protocol_features,nearcore/performance_stats,nearcore/memory_stats
-	cargo build -p runtime-params-estimator --release --features nightly_protocol,nightly_protocol_features,nearcore/nightly_protocol,nearcore/nightly_protocol_features,nearcore/performance_stats,nearcore/memory_stats
-	cargo build -p genesis-populate --release --features nearcore/nightly_protocol,nearcore/nightly_protocol_features,nearcore/performance_stats,nearcore/memory_stats
+	cargo build -p store-validator --release --features nearcore/nightly,nearcore/performance_stats,nearcore/memory_stats
+	cargo build -p runtime-params-estimator --release --features nightly,nearcore/nightly,nearcore/performance_stats,nearcore/memory_stats
```

Comment:
cc @jakmeier I think we can remove runtime-params-estimator from releases altogether perhaps? (unrelated to the PR)

Reply:
Sounds right, yeah. Created https://github.com/near/nearcore/pull/6785